doc/API.md

```diff
@@ -311,6 +311,7 @@ Additionally, there are a number or properties that allow configuration of diffe
 | `absolutePaths` | `false` | Use absolute paths when creating element selectors
 | `iframes`       | `true`  | Tell axe to run inside iframes
 | `elementRef`    | `false` | Return element references in addition to the target
+| `restoreScroll` | `false` | Scrolls elements back to before axe started
```

Comment:
@dylanb @marcysutton Based on previous discussion, I think we concluded to leave it on false (as it is now). Please approve if you think this is ok.